packages/core/system/src/filter-html-attributes/filter-html-attributes.ts

```
@@ -0,0 +1,25 @@
+import _ from "lodash";
```

ESLint

[error] Unable to resolve path to module 'lodash'.

(import-x/no-unresolved)